packages/server/src/tests/utilities/api/base.ts

```diff
@@ -236,3 +239,24 @@ export abstract class TestAPI {
     ).body
   }
 }
+
+export abstract class PublicAPI extends TestAPI {
+  prefix = "/api/public/v1"
```

Comment:
Chosen specifically so that the URL we pass in children of this class will match up with the URL specified in the routes files.